### PR TITLE
Update Boilerplate

### PR DESCRIPTION
**Pull request by RobLoach**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,14 +1,17 @@
 {
-  "name": "jstransformer-hamlet",
-  "version": "0.1.0",
-  "description": "Hamlet support for JSTransformers",
+  "name": "jstransformer-foo",
```
Comment (ForbesLindesay) [inline package.json]: This should still be `jstransformer-hamlet`

File: index.js [hunk 2]
```diff
@@ -4,14 +4,8 @@ var hamlet = require('hamlet');
 var extend = require('extend-shallow');
```
Comment (ForbesLindesay) [inline index.js]: Looks like extend-shallow is no longer used?

File: package.json [hunk 3]
```diff
@@ -1,14 +1,17 @@
...
+  "description": "Hamlet support for JSTransformers.",
   "keywords": [
-    "hamlet",
     "jstransformer"
   ],
   "files": [
     "index.js"
   ],
+  "dependencies": {
+    "extend-shallow": "^2.0.1",
```
Comment (ForbesLindesay) [inline package.json]: Looks like extend-shallow is no longer used?

File: package.json [hunk 4]
```diff
@@ -16,11 +19,16 @@
...
-  "dependencies": {
-    "extend-shallow": "^2.0.1",
-    "hamlet": "^0.3.3"
-  }
+  "engines": {
+    "node": ">=0.10.36"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/jstransformers/jstransformer-foo.git"
```
Comment (ForbesLindesay) [inline package.json]: This should still be `jstransformer-hamlet`

**Comment by ForbesLindesay** [comment]
Couple of nits, otherwise looks good.

**Comment by RobLoach** [comment]
:+1:

**Pull request merged**